src/cmd/manage_nsfs.js

```diff
@@ -0,0 +1,452 @@
+                                                        Get bucket and account details from json file                
+
+    # required for add, replace 
+    --name <name>               (default none)          Set the name for account/bucket.
+    --email <email>             (default none)          Set the email for account/bucket.
+    --uid <uid>                 (default as process)    Send requests to the Filesystem with uid.
+    --gid <gid>                 (default as process)    Send requests to the Filesystem with gid.
+    --secret_key <key>          (default none)          The secret key pair for the access key.
+    --new_buckets_path <dir>    (default none)          Set the root of the filesystem where each subdir is a bucket.
+    
+    # required for add, replace and delete
+    --access_key <key>          (default none)          Authenticate incoming requests for this access key only (default is no auth).
+    --config_root <dir>         (default none)          Configuration files path for Noobaa standalon NSFS.
+`;
+
+const BUCKET_OPTIONS = `
+Bucket Options:
+
+    # Read Bucket details from json file, no need to mention all the propery one by one in CLI
+    --from_file <dir>           (default none)          Set buckt/account schema full path.
+                                                        Get bucket and account details from json file
+    # required for add, replace 
+    --email <email>             (default none)          Set the email for account/bucket.
+    --path <dir>                (default none)          Set the bucket path.
+
+    # required for add, replace and delete
+    --name <name>               (default none)          Set the name for account/bucket.
+    --config_root <dir>         (default none)          Configuration files path for Noobaa standalon NSFS.
+`;
+
+function print_account_usage() {
+    console.warn(HELP);
+    console.warn(USAGE.trimStart());
+    console.warn(ARGUMENTS.trimStart());
+    console.warn(ACCOUNT_OPTIONS.trimStart());
+    process.exit(1);
+}
+
+function print_bucket_usage() {
+    console.warn(HELP);
+    console.warn(USAGE.trimStart());
+    console.warn(ARGUMENTS.trimStart());
+    console.warn(BUCKET_OPTIONS.trimStart());
+    process.exit(1);
+}
+
+async function check_and_create_config_dirs(config_root) {
+    const bucket_dir_stat = await fs_utils.file_exists(config_root);
+    if (!bucket_dir_stat) {
+        await fs_utils.create_path(config_root);
+        await fs_utils.create_path(config_root + '/buckets');
+        await fs_utils.create_path(config_root + '/accounts');
+        return true;
+    }
+    return false;
+}
+
+async function main(argv = minimist(process.argv.slice(2))) {
+    try {
+        const resourse_type = argv._[0] || '';
+        if ((argv.help || argv.h) && resourse_type === 'account') return print_account_usage();
+        if ((argv.help || argv.h) && resourse_type === 'bucket') return print_bucket_usage();
+        if (argv.debug) {
```

Comment:
so, do we support updating the debug level through this CLI? might be a good idea, we just need it documented and tested

Reply:
@romayalon When I thought about it, it doesn't look good to have debug level changes in this CLI. WDYT? Should I remove it?

Reply:
you can remove it from this PR and we should think how to let users do that

Reply:
done